resource/pom.xml

```diff
@@ -75,6 +75,11 @@
             <version>3.1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>42.4.0</version>
+        </dependency>
```

Comment:
Move versions to a separate section (properties).

Reply:
Done